### PR TITLE
MULE-10656: Minor changes in the FTP TestHarness to be able to use it…

### PR DESCRIPTION
**Pull request by anosenzo**
… in the distributions tests

### CODE REVIEW

File: extensions/ftp/src/test/java/org/mule/extension/FtpTestHarness.java [hunk 1]
```diff
@@ -6,10 +6,9 @@
  */
 package org.mule.extension;
 
-import org.mule.extension.ftp.api.FtpFileAttributes;
-
 import org.junit.rules.ExpectedException;
 import org.junit.rules.TestRule;
+import org.mule.extension.ftp.api.FtpFileAttributes;
```
Comment (aiannucci) [inline extensions/ftp/src/test/java/org/mule/extension/FtpTestHarness.java]: Fix import ordering

File: extensions/ftp/src/test/java/org/mule/extension/ftp/ClassicFtpTestHarness.java [hunk 2]
```diff
@@ -24,8 +23,9 @@
 import java.time.ZoneId;
 import java.util.Calendar;
 
-import org.apache.commons.net.ftp.FTPFile;
-import org.junit.rules.TestRule;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
```
Comment (aiannucci) [inline extensions/ftp/src/test/java/org/mule/extension/ftp/ClassicFtpTestHarness.java]: Fix import ordering

File: extensions/ftp/src/test/java/org/mule/extension/ftp/SftpTestHarness.java [hunk 3]
```diff
@@ -30,8 +24,14 @@
...
-import org.junit.rules.TemporaryFolder;
-import org.junit.rules.TestRule;
+import static java.util.stream.Collectors.toList;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mule.extension.file.common.api.FileWriteMode.APPEND;
+import static org.mule.extension.file.common.api.FileWriteMode.OVERWRITE;
+import static org.mule.functional.util.sftp.SftpServer.PASSWORD;
+import static org.mule.functional.util.sftp.SftpServer.USERNAME;
```
Comment (aiannucci) [inline extensions/ftp/src/test/java/org/mule/extension/ftp/SftpTestHarness.java]: Fix import ordering

File: extensions/ftp/src/test/java/org/mule/extension/FtpTestHarness.java [hunk 4]
```diff
@@ -95,6 +94,11 @@
   String[] getFileList(String path) throws Exception;
 
   /**
+   * Returns Server Port
+   */
```
Comment (aiannucci) [inline extensions/ftp/src/test/java/org/mule/extension/FtpTestHarness.java]: Add @ return clause

File: extensions/ftp/src/test/java/org/mule/extension/ftp/SftpTestHarness.java [hunk 5]
```diff
@@ -182,14 +182,22 @@ public boolean changeWorkingDirectory(String path) throws Exception {
    */
   @Override
   public String[] getFileList(String path) throws Exception {
-    List<String> files = sftpClient.list(path).stream().map(FileAttributes::getPath).collect(toList());
+    List<String> files = sftpClient.list(path).stream().map(FileAttributes::getName).collect(toList());
```
Comment (aiannucci) [inline extensions/ftp/src/test/java/org/mule/extension/ftp/SftpTestHarness.java]: Isn't this going to break existing tests?

Comment (marianogonzalez) [inline extensions/ftp/src/test/java/org/mule/extension/ftp/SftpTestHarness.java]: this change is not equivalent

**Comment by aiannucci** [comment]
@marianogonzalez please approve this PR if it's OK so it can be merged.

**Pull request merged**